core/sis-feature/src/test/java/org/apache/sis/feature/builder/FeatureTypeBuilderTest.java

```diff
@@ -344,4 +347,95 @@ private static void assertPropertyEquals(final String name, final Class<?> value
         }
         assertEquals("valueClass", valueClass, ((DefaultAttributeType<?>) property).getValueClass());
     }
+
+    @Ignore("Obviously a bug.")
+    @Test
+    public void testToStringInternalIsNotNull() {
+        FeatureTypeBuilder featureTypeBuilder = new FeatureTypeBuilder(null);
+        DefaultFeatureType[] defaultFeatureTypeArray = new DefaultFeatureType[6];
+        featureTypeBuilder.setSuperTypes(defaultFeatureTypeArray);
+        StringBuilder stringBuilder = new StringBuilder("");
+        featureTypeBuilder.toStringInternal(stringBuilder);
+
+        assertNotNull(stringBuilder.toString());
+    }
+
+    @Test
+    public void testBuildThrowsRuntimeException() {
+        FeatureTypeBuilder featureTypeBuilder = new FeatureTypeBuilder();
+        FeatureTypeBuilder featureTypeBuilderTwo = featureTypeBuilder.setIdentifierDelimiters("[RML1*!JGC`fF+|u", "[RML1*!JGC`fF+|u", "[RML1*!JGC`fF+|u");
+        featureTypeBuilderTwo.identifierCount = 1;
+
+        try {
+            featureTypeBuilder.build();
+            fail("Expecting exception: RuntimeException");
+        } catch(RuntimeException e) {
+            assertEquals(FeatureTypeBuilder.class.getName(), e.getStackTrace()[0].getClassName());
+        }
+    }
+
+    @Test
+    public void testGetNameSpaceReturningCharSequenceWhereLengthIsPositive() {
+        FeatureTypeBuilder featureTypeBuilder = new FeatureTypeBuilder();
+        FeatureTypeBuilder featureTypeBuilderTwo = featureTypeBuilder.setNameSpace("java.awt.Point[x=98,y=98]");
+
+        assertEquals("java.awt.Point[x=98,y=98]", featureTypeBuilderTwo.getNameSpace());
+    }
+
+    @Test
+    public void testGetNameSpaceReturningNull() {
+        DefaultNameFactory defaultNameFactory = new DefaultNameFactory();
+        GeometryLibrary geometryLibrary = GeometryLibrary.JAVA2D;
+        FeatureTypeBuilder featureTypeBuilder = new FeatureTypeBuilder(defaultNameFactory, geometryLibrary, null);
+
+        assertNull(featureTypeBuilder.getNameSpace());
+    }
+
+    @Test
+    public void testSetAbstractReturningFeatureTypeBuilderWhereIsAbstractIsTrue() {
+        FeatureTypeBuilder featureTypeBuilder = new FeatureTypeBuilder(null);
+        featureTypeBuilder.setAbstract(true);
+        StringBuilder stringBuilder = new StringBuilder();
+        featureTypeBuilder.toStringInternal(stringBuilder);
+
+        assertTrue(featureTypeBuilder.isAbstract());
+    }
+
+    @Test
+    public void testSetAbstractReturningFeatureTypeBuilderWhereIsAbstractIsFalse() {
+        FeatureTypeBuilder featureTypeBuilder = new FeatureTypeBuilder(null);
+        FeatureTypeBuilder featureTypeBuilderTwo = featureTypeBuilder.setAbstract(false);
+
+        assertFalse(featureTypeBuilderTwo.isDeprecated());
+    }
+
+    @Test
+    public void testSetAll() {
```

Comment:
It is not clear to me what this method is testing that other methods do not already test.